plugins/TwoFactorAuth/lang/en.json

```
@@ -2,50 +2,50 @@
-        "SetupAuthenticatorOnDeviceStep2": "Next, open the app and scan the below bar code with the two-factor authentication app on your phone. If you can’t scan the barcode, %1$senter this code%2$s instead.",
-        "SetupBackupRecoveryCodes": "Please backup your recovery codes using one of the above methods before continuing the two-factor authentication setup.",
+        "SetupAuthenticatorOnDevice": "Set up an OTP app on your device",
+        "SetupAuthenticatorOnDeviceStep1": "Install an OTP app, for example:",
+        "SetupAuthenticatorOnDeviceStep2": "Next, open the app and scan the barcode below with the OTP app on your mobile device. %1$sEnter this code%2$s if you can't scan the barcode.",
+        "SetupBackupRecoveryCodes": "Please save your recovery codes securely using one of the methods above before continuing.",
         "RecoveryCodes": "Recovery codes",
-        "RecoveryCodesExplanation": "You can use recovery codes to access your account when you cannot receive two-factor authentication codes, for example when you don't have your mobile device with you.",
-        "RecoveryCodesSecurity": "Please treat your recovery codes with the same level of security as you would your password!",
-        "RecoveryCodesAllUsed": "All recovery codes have been used, it is highly recommended you regenerate your recovery codes.",
-        "RecoveryCodesRegenerated": "Recovery codes have been regenerated. Make sure to download or print the newly generated codes.",
+        "RecoveryCodesExplanation": "You can use recovery codes to access your account when you cannot receive two-factor authentication codes (for example when you don't have your mobile device with you).",
+        "RecoveryCodesSecurity": "Please protect your recovery codes like a password.",
+        "RecoveryCodesAllUsed": "All recovery codes have been used. You must generate new ones.",
+        "RecoveryCodesRegenerated": "Recovery codes generated. Make sure to download or print out the newly generated codes.",
         "GenerateNewRecoveryCodes": "Generate new recovery codes",
-        "GenerateNewRecoveryCodesInfo": "When you generate new recovery codes, your old codes won’t work anymore. Make sure to download or print your new codes.",
+        "GenerateNewRecoveryCodesInfo": "Generating new recovery codes means your old ones won't work anymore. Download or print out your new codes.",
         "RequireTwoFAForAll": "Require two-factor authentication for everyone",
-        "RequireTwoFAForAllInformation": "When enabled, every user has to enable two factor authentication. Enforcing 2FA will require all users to have access to a device where they can install an authenticator app. You can only enable this if you already have two factor authentication set up yourself."
+        "RequireTwoFAForAllInformation": "Requires other users also log in with 2FA by using an OTP app on their mobile device to log in. (As a prerequisite, you must use 2FA yourself.)"
```

isn't the second `log in` a not needed duplication?
```suggestion
        "RequireTwoFAForAllInformation": "Requires other users also log in with 2FA by using an OTP app on their mobile device. (As a prerequisite, you must use 2FA yourself.)"
```